tests/codegen/llvm/map_key_struct.ll

```diff
@@ -18,12 +18,12 @@ entry:
   %probe_read_kernel = call i64 inttoptr (i64 113 to i64 ([4 x i8]*, i32, i64)*)([4 x i8]* %"@x_key", i32 4, i64 %arg0)
   %4 = bitcast i64* %"@x_val" to i8*
   call void @llvm.lifetime.start.p0i8(i64 -1, i8* %4)
-  store i64 44, i64* %"@x_val"
+  store i64 %arg0, i64* %"@x_val"
```

Comment:
okay, this part looks substantially different. might explain the runtime test failure.